proto/storage/declarative_config_health.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+
+package storage;
+
+message DeclarativeConfigHealth {
+    enum Status {
+        UNHEALTHY     = 0;
+        HEALTHY       = 1;
+    }
+
+    string id                                     = 1 [(gogoproto.moretags) = 'sql:"pk"'];
+    string name                                   = 2;
+    Status status                                 = 4;
+    string error_message                          = 5;
+    string resource_name                          = 6;
```

Comment:
In addition to the resource name, we can also think about adding a type here to infer a) which resource this is to potentially create a link to the resource within the UI b) distinguish between config map and other objects.

WDYT?

Reply:
Do we envision any use cases where we may query on some of these other fields to avoid `walks` or is this data sufficiently small so the `walks` are fine.

Reply:
I suppose this data should be sufficiently small (currently within the MVP we don't not more than 100 configs), it'd be more of a future-type-thing if that makes sense.

Reply:
Sounds fair.  that is something I'm looking at for what I'm working on so I figured I would ask since it is fresh in my mind.  Easy enough to promote something to a column for search when the need arises.